src/homework/homework7/DividingOnZeroException.java

```diff
@@ -0,0 +1,8 @@
+package homework.homework7;
+
+public class DividingOnZeroException extends Exception {
+
+    DividingOnZeroException() {
```

Comment:
No necessity to create your own exception because there is ArithmeticException for it